be/src/exec/except_hash_set.cpp

```diff
@@ -103,8 +104,10 @@ void ExceptHashSet<HashSet>::deserialize_to_columns(KeyVector& keys, Columns& ke
             }
         }
 
-        key_column->deserialize_and_append_batch(keys, chunk_size);
+        TRY_CATCH_BAD_ALLOC(key_column->deserialize_and_append_batch(keys, chunk_size));
```

Comment:
Is it possible to expand the scope of the Try-Catch block? The `update_if_overflow` maybe also allocate a large amount of memory.

Reply:
fixed